synapse/lib/cell.py

```diff
@@ -884,6 +884,11 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'type': 'object',
             'hidecmdl': True,
         },
+        'https:parse:xheaders': {
```

Comment:
"xheaders" feels too generic to me given what this is specifically enabling, unless you were planning for all future x- headers to also be enabled by this?

Reply:
renamed per discussion